src/main/resources/lang/en_us.json

```diff
@@ -0,0 +1,97 @@
+{
+	"language": "English",
+
+	"generic.mapping_format.enigma_file": "Enigma File",
+	"generic.mapping_format.enigma_directory": "Enigma Directory",
+	"generic.mapping_format.tiny_v2": "Tiny v2",
+	"generic.mapping_format.tiny_file": "Tiny File",
+	"generic.mapping_format.srg_file": "SRG File",
+	"generic.mapping_format.proguard": "Proguard",
+	"generic.type.methods": "Methods",
+	"generic.type.fields": "Fields",
+	"generic.type.parameters": "Parameters",
+	"generic.type.classes": "Classes",
+
+	"menubar.file": "File",
+	"menubar.file.jar.open": "Open Jar...",
+	"menubar.file.jar.close": "Close Jar",
+	"menubar.file.mappings.open": "Open Mappings...",
+	"menubar.file.mappings.save": "Save Mappings",
+	"menubar.file.mappings.save_as": "Save Mappings As...",
+	"menubar.file.mappings.close": "Close Mappings",
+	"menubar.file.mappings.drop": "Drop Invalid Mappings",
+	"menubar.file.export.source": "Export Source...",
+	"menubar.file.export.jar": "Export Jar...",
+	"menubar.file.stats": "Mapping Stats...",
+	"menubar.file.stats.generate": "Generate Stats",
+	"menubar.file.exit": "Exit",
+	"menubar.view": "View",
+	"menubar.view.themes": "Themes",
+	"menubar.view.themes.default": "Default",
+	"menubar.view.themes.darcula": "Darcula",
+	"menubar.view.themes.system": "System",
+	"menubar.view.themes.none": "None (JVM Default)",
+	"menubar.view.languages": "Languages",
+	"menubar.view.search": "Search",
+	"menubar.help": "Help",
+	"menubar.help.about": "About",
+	"menubar.help.about.ok": "Ok",
+	"menubar.help.github": "Github Page",
+
+	"popup_menu.rename": "Rename",
+	"popup_menu.javadoc": "Edit Javadoc",
+	"popup_menu.inheritance": "Show Inheritance",
+	"popup_menu.implementations": "Show Implementations",
+	"popup_menu.calls": "Show Calls",
+	"popup_menu.calls.specific": "Show Calls (Specific)",
+	"popup_menu.declaration": "Go to Declaration",
+	"popup_menu.back": "Go back",
+	"popup_menu.forward": "Go forward",
+	"popup_menu.mark_deobfuscated": "Mark as deobfuscated",
+	"popup_menu.reset_obfuscated": "Reset to obfuscated",
+
+	"panel.classes.obfuscated": "Obfuscated Classes",
+	"panel.classes.deobfuscated": "De-obfuscated Classes",
+	"panel.identifier": "Identifier Info",
+	"panel.identifier.none": "No identifier selected",
+	"panel.identifier.variable": "Variable",
+	"panel.identifier.field": "Field",
+	"panel.identifier.method": "Method",
+	"panel.identifier.constructor": "Constructor",
+	"panel.identifier.class": "Class",
+	"panel.identifier.type_descriptor": "TypeDescriptor",
+	"panel.identifier.method_descriptor": "MethodDescriptor",
+	"panel.identifier.modifier": "Modifier",
+	"panel.identifier.index": "Index",
+	"panel.editor.class.decompiling": "(decompiling...)",
+	"panel.editor.class.not_found": "Unable to find class:",
+	"panel.tree.inheritance": "Inheritance",
+	"panel.tree.implementations": "Implementations",
+	"panel.tree.calls": "Call Graph",
```

Comment:
`info_panel` would be more specific, in case more panels are added in the future.